example/main.cpp

```diff
@@ -21,8 +21,8 @@ int main(int argc, char *argv[]) {
 
     spdlog::set_default_logger(vulkan_renderer_log);
 
-    spdlog::debug("Inexor vulkan-renderer, BUILD " + std::string(__DATE__) + ", " + __TIME__);
-    spdlog::debug("Parsing command line arguments.");
+    spdlog::trace("Inexor vulkan-renderer, BUILD " + std::string(__DATE__) + ", " + __TIME__);
```

Comment:
Should stay debug imho and include more meta information. like GIT_SHA

Reply:
That will be dealt with in that pull request which deals with the build metadata.